### PR TITLE
Fix/update checker

### PR DESCRIPTION
**Pull request by darbyjack**

### CODE REVIEW

File: src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt [hunk 1]
```diff
@@ -36,45 +32,32 @@ object UpdateChecker
 		}
 		
 		
-		val old = version
 		val new = try
```
Comment (Sxtanna) [inline src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt:26]: this is doing... nothing?

File: src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt [hunk 2]
```diff
@@ -36,45 +32,32 @@ object UpdateChecker
...
 		}
 		
-		if (old == new)
+		if (version == new)
 		{
 			return UpdateResult.UP_TO_DATE
 		}
 		
-		val oldVersion = old.split('.').mapNotNull(String::toIntOrNull)
+		val oldVersion = version.split('.').mapNotNull(String::toIntOrNull)
```
Comment (Sxtanna) [inline src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt]: why the variable name change?

File: src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt [hunk 3]
```diff
@@ -1,16 +1,12 @@
...
 import org.bukkit.plugin.Plugin
 import java.net.URL
 
 object UpdateChecker
 {
 	
-	private const val API = "https://api.spiget.org/v2/resources/%d/versions?size=1&sort=-releaseDate"
-	private val LIST_TYPE = object : TypeToken<List<Version>>()
-	{}.type
+	private const val API = "https://api.spigotmc.org/legacy/update.php?resource=987"
```
Comment (Sxtanna) [inline src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt]: you removed the format variable `%d` making this hard coded for 987.
(not an issue, just a behaviour change you should be aware of considering the front end hasnt changed)

File: src/test/kotlin/me/clip/voteparty/UpdateCheckerTests.kt [hunk 4]
```diff
@@ -10,7 +10,7 @@ object UpdateCheckerTests
 	@Test
 	internal fun testUpdate()
 	{
-		val result = UpdateChecker.check("1.14.0", 987)
+		val result = UpdateChecker.check("2.20", 987)
```
Comment (Sxtanna) [inline src/test/kotlin/me/clip/voteparty/UpdateCheckerTests.kt:13]: the passed value 987 does nothing

File: src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt (hunk 3 above)
Comment (darbyjack) [reply src/main/kotlin/me/clip/voteparty/util/UpdateChecker.kt]: Unhardcoded it.

**Review by Sxtanna — APPROVED** [review]

**Pull request merged**